src/frontend/next/src/pages/myfeeds.tsx

```diff
@@ -0,0 +1,216 @@
+import { useState, useEffect, useContext, FC, FormEvent } from 'react';
+import { useRouter } from 'next/router';
+import { Box, Card, Container, Grid, IconButton, Typography } from '@material-ui/core';
```

Comment:
We need a rebase on this to pickup the changes in package.json, so this can be resolved.